hamilton/plugins/pandas_extensions.py

```diff
@@ -8,6 +8,9 @@
 from pathlib import Path
 from typing import Any, Callable, Collection, Dict, Iterator, List, Optional, Tuple, Type, Union
 
+import fsspec
+import pyarrow.fs
```

Comment:
These need to be moved after the pandas import check.

I believe pandas depends on fsspec, so it should be there. Otherwise pyarrow might not. So we should do an import check for it, and if it's not there use `Type` or something -- since it's only used as a type annotation.

Reply:
actually it seems like fsspec might not be -- so we should check for it and use `Type` if it's not installed.